apps/api/tsconfig.app.json

```
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
-    "module": "commonjs",
+    "module": "Preserve",
```

⚙️ Not sure if this is case sensitive but [official docs](https://www.typescriptlang.org/tsconfig/#module) mention lower case only. Applicable  in multiple places.
```suggestion
    "module": "preserve",
```

---

It's case insensitive. VSCode autocompletes the PascalCase versions.

---

I see, strange that the official docs show a different case than what is suggested by VSCode.